Import the service rules (#5)
DFE-Digital/ecf2: documentation/service-rules/appropriate-body-users.md

> # Appropriate body user

Reviewer: ```suggestion
# Appropriate body users
```

### **Get access to the TRA's appropriate body sign in**

Access to the AB portal, and user management, is administered via DfE
sign-in.

* 🔒 Within DfE Sign-In, each organisation has a nominated approver,
and that person (rather than DfE personnel) is responsible for
adminstering user requests for their organisation. This keeps the data
safe from people within organisations that should not have access to
it.
---
ABs as an organization are added or removed as needed through liaising
with the induction data collections team in the TRA via email.

* 🔒 This means organisations can only get access that are legitimately
appropriate bodies. Policy outlining who can be an appropriate body:
[Appropriate bodies guidance - section
3:1](https://assets.publishing.service.gov.uk/media/661d459fac3dae9a53bd3de6/Appropriate_bodies_guidance_induction_and_the_early_career_framework.pdf)
---
### **View records for early career teachers that are already registered**

They can view historical records for ECTs they have claimed in a list on
their Appropriate body account homepage.

They can also view their appropriate body ID. This is referred to later
as an employer ID.\
📊

### 
---

> ### **Individually claim an ECT for their induction status with the TRA**

Reviewer: ```suggestion
### Individually claim an ECT for their induction status with the TRA
```

#### QTS status

-   Qualified teacher status

-   QTS date

-   Date of Partial Recognition

#### Induction status

-   Induction status (In progress/Not yet
    completed/Extended/Passed/Failed)

-   Eligible to complete a one-year induction period under transition
    arrangements

* 📜 This part was related to temporary provisions when the ECF came
in. If someone, as of 1 September 2021, had started but not completed
induction, they had until 31 August 2023 to complete induction under the
'old regs'. TRA populated this themselves, based on data they had. ABs
cannot edit it.

* The transition period is over, so this is now a 'no' for everyone.
---
There are provisions in the regulations for ABs to offer a reduced
induction. Appropriate bodies have discretion to reduce the length of
the induction period to a minimum of one term (based on a school year of
three terms) to recognise previous teaching experience.

Induction history is shown, as subsection of induction status, if the
ECT has a previous appropriate body(s) than their current one.

* 🙋 This section is so that ABs can track where their ECTs have been
and how much induction they have served elsewhere.

> ----

Reviewer: ```suggestion
---
```

as an employer ID.

---
### Individually claim an ECT for their induction status with the TRA

The process of 'claiming' an ECT refers to an AB telling the TRA which
ECTs they are quality assuring the induction of. 

The appropriate body needs to enter both the date of birth and the TRN
number for the ECT record to show up and be claimed.

* 🔒 The AB needs both the TRN and DOB as a data privacy measure to
ensure that the AB is only searching for ECTs they already have links
with. This search locates the record in the database of qualified
teachers (DQT).

If there is no TRN and DOB match, then an error message appears. The AB
would then liaise with our induction data collections team or the teacher themselves
as needed to verify data.

The record will still show but have a blank field in the QTS date
section if they don't have QTS yet.

If the ECT has previously failed induction, they cannot do induction
again.

* 📜 This is due to policy restricting those who have failed from
reapplying to practice teaching. [AB
guidance details this here.](https://assets.publishing.service.gov.uk/media/6629237f3b0122a378a7e6ef/Induction_for_early_career_teachers__England__statutory_guidance_.pdf)

> 

Reviewer 2: ```suggestion

If an early career teacher did their initial teacher training (ITT) through an accredited ITT provider who is also an AB, the school cannot appoint that appropriate body for that teacher.

ECTs may be serving their induction at an AB. For example, they may work at a teaching school hub that is also an AB. If this is the case, the school cannot appoint that AB for that teacher.

[Find out more in section 2 of the statutory guidance on induction for early career teachers.](https://www.gov.uk/government/publications/induction-for-early-career-teachers-england?)

```

When an appropriate body user tries to view an ECT record to claim them,
if the ECT is already claimed by another appropriate body, it would
state they were 'in progress' and the appropriate body they were in
progress with.

* 💻 They are unable to change this on the service so the appropriate
body user would have to reach out to the other appropriate body or the
TRA induction support team to get them to release the ECT.

* 📜 Sometimes, these appropriate bodies (particularly local
authorities due to policy changes) no longer exist so the induction
support team would help the ABs to navigate this.

---

### Claiming an ECT

When they are claiming an ECT, the appropriate body user needs to state:

1.  When the induction began for that ECT with that appropriate body

2.  What type of induction programme they are doing (FIP, CIP, DIY)

On 1, there is validation that makes sure the date given isn't
before the date QTS was recorded. This is because induction should only
start after they received QTS.

There are some discrepancies in the rules on the portal. Future dates
aren't allowed in the web update, but can get through the bulk
import upload.

> The end date must be before a start date for an induction period.

Reviewer: ```suggestion
The end date of an induction period must be after the start date for an induction period.
```

safe from people within organisations that should not have access to
it.
---
ABs as an organization are added or removed as needed through liaising
with the induction data collections team in the TRA via email.

* 🔒 This means organisations can only get access that are legitimately
appropriate bodies. Policy outlining who can be an appropriate body:
[Appropriate bodies guidance - section
3:1](https://assets.publishing.service.gov.uk/media/661d459fac3dae9a53bd3de6/Appropriate_bodies_guidance_induction_and_the_early_career_framework.pdf)
---
### View records for early career teachers that are already registered

They can view historical records for ECTs they have claimed in a list on
their Appropriate body account homepage.

They can also view their appropriate body ID. This is referred to later
as an employer ID.

---
### Individually claim an ECT for their induction status with the TRA

The process of 'claiming' an ECT refers to an AB telling the TRA which
ECTs they are quality assuring the induction of. 

The appropriate body needs to enter both the date of birth and the TRN
number for the ECT record to show up and be claimed.

> * 🔒 The AB needs both the TRN and DOB as a data privacy measure to

Reviewer: Why?

ensure that the AB is only searching for ECTs they already have links
with. This search locates the record in the database of qualified
teachers (DQT).

If there is no TRN and DOB match, then an error message appears. The AB
would then liaise with our induction data collections team or the teacher themselves
as needed to verify data.

The record will still show but have a blank field in the QTS date
section if they don't have QTS yet.

If the ECT has previously failed induction, they cannot do induction
again.

* 📜 This is due to policy restricting those who have failed from
reapplying to practice teaching. [AB
guidance details this here.](https://assets.publishing.service.gov.uk/media/6629237f3b0122a378a7e6ef/Induction_for_early_career_teachers__England__statutory_guidance_.pdf)

> If an early career teacher did their initial teacher training (ITT) through an accredited ITT provider who is also an AB, the school cannot appoint that appropriate body for that teacher.

Reviewer: Kerry added this. I think given we don't really need to know now, we leave for now, but happy for someone to check.
